packages/google-pay-integration/.eslintrc.json

```diff
@@ -8,7 +8,8 @@
                 "@typescript-eslint/naming-convention": "off",
                 "@typescript-eslint/no-unsafe-argument": "off",
                 "@typescript-eslint/no-unsafe-assignment": "off",
-                "@typescript-eslint/consistent-type-assertions": "off"
+                "@typescript-eslint/consistent-type-assertions": "off",
+                "@typescript-eslint/no-unnecessary-condition": "off"
```

Comment:
Why do disable this rule?

Reply:
Because of this method on the line 46, es-lint says that value is always truthy, but it can be also false.
<img width="731" alt="Screenshot 2024-10-23 at 16 25 47" src="https://github.com/user-attachments/assets/5014b471-5c34-4ecd-bd98-318d21887b08">

Reply:
What if we move this checking to 50 line?

Reply:
I believe it's better to ignore just one line, and do not make a rule for the all files.